libr/core/cmd.c

```diff
@@ -511,6 +512,7 @@ static int cmd_alias(void *data, const char *input) {
 	char *q = strchr (buf, ' ');
 	char *def = strchr (buf, '=');
 	char *desc = strchr (buf, '?');
```

Comment:
The strchr seems like a very weak way to detect this, because this way the user cant echo a ? I think

Reply:
I wanted to do a minimal change.
I can refactor this if you prefer.